tasks/src/main/scala/dagr/tasks/picard/SamToFastq.scala

```diff
@@ -46,17 +46,26 @@ class SamToFastq(in: PathToBam,
                  fastq2: Option[PathToFastq] = None,
                  interleave : Boolean = true,
                  clippingAttribute: Option[String] = None,
-                 clippingAction: Option[String] = None) extends PicardTask with Pipe[SamOrBam,Fastq] {
+                 clippingAction: Option[String] = None,
+                 clippingMinReadLength: Int = 25) extends PicardTask with Pipe[SamOrBam,Fastq] {
```

Comment:
I'd rather leave it like this - mostly because I think that other than for backwards compatibility reasons I would have changed `SamToFastq` in Picard to have this behaviour by default.  I.e. I think it would be too easy to forget and not supply a `clippingMinReadLength`, and I'd rather not have that be a runtime error.